packages/xo-web/src/common/render-xo-item.js

```diff
@@ -251,6 +266,7 @@ export const Sr = decorate([
 ])
 
 Sr.propTypes = {
+  allocationStrategy: PropTypes.bool,
```

Comment:
A variable with the name `allocationStrategy` seem like it contains the selected allocation. 

Maybe `showAllocationStrategy` which the possible response is `Yes` or `No`.